pytest_watch/__main__.py

```diff
@@ -8,12 +8,14 @@
 :license: MIT, see LICENSE for more details.
 """
 
-import os
-import sys
 
+def run_cli():
+    import os
+    import sys
 
-if __name__ == '__main__':
     sys.path.append(os.path.dirname(__file__))
 
     from pytest_watch.command import main
-    main()
+    main(argv=sys.argv[1:])
```

Comment:
I think `main()` with no args is more aesthetic. The default is to use `argv[1:]` and adding that here adds noise.

Could you change it back?

If your rationale was "I couldn't tell what main's default were", a better docstring for `main` could address that instead of more code.